internal/tiltfile/files.go

```diff
@@ -200,17 +200,18 @@ func (s *tiltfileState) skylarkReadFile(thread *starlark.Thread, fn *starlark.Bu
 		return nil, err
 	}
 
-	return newBlob(string(bs)), nil
+	return newBlob(string(bs), fmt.Sprintf("file: %s", path)), nil
```

Comment:
path is probably not right here...i think you want to pull the path off `p`, in case path is a non-string?

Reply:
good catch, thanks!